third_party/istio-latest

```diff
@@ -1 +1 @@
-istio-1.4.4
\ No newline at end of file
+istio-1.5.0/
```

Comment:
do we need to include `\` ?

Reply:
This is a symlink, the content is automatically generated by git I think.